_sass/footer.scss

```diff
@@ -1,2 +1,45 @@
 /* Footer Styles */
 
+footer {
+	max-height: 265px;
+	color: $navy;
+}
+
+.wrapper {
+	max-height: 190px;
```

Comment:
Let's delete this specific max-height and instead use top and bottom padding of 2rem. This max-height calculation produced a bug where there was no padding or margin above it
<img width="683" alt="Screenshot 2020-04-30 19 29 21" src="https://user-images.githubusercontent.com/32233894/80777914-921d6800-8b1b-11ea-8f1d-5379612d5bae.png">

Reply:
Removed max-height and added top and bottom padding. After a few manual visual tests, we might want to consider changing the padding on mobile views.